plugins/woocommerce/src/Blocks/BlockTypes/Checkout.php

```diff
@@ -350,9 +356,15 @@ protected function enqueue_data( array $attributes = [] ) {
 		$this->asset_data_registry->register_page_id( isset( $attributes['cartPageId'] ) ? $attributes['cartPageId'] : 0 );
 		$this->asset_data_registry->add( 'isBlockTheme', wc_current_theme_is_fse_theme(), true );
 
-		$pickup_location_settings = get_option( 'woocommerce_pickup_location_settings', [] );
+		$pickup_location_settings = get_option(
+			'woocommerce_pickup_location_settings',
+			[
+				'enabled' => false,
+				'title'   => __( 'Local Pickup', 'woocommerce' ),
+			]
+		);
 		$this->asset_data_registry->add( 'localPickupEnabled', wc_string_to_bool( $pickup_location_settings['enabled'] ?? 'no' ), true );
-		$this->asset_data_registry->add( 'localPickupText', wc_clean( $pickup_location_settings['title'] ) ?? __( 'Local Pickup', 'woocommerce' ), true );
+		$this->asset_data_registry->add( 'localPickupText', wc_clean( $pickup_location_settings['title'] ?? '' ) ?? __( 'Local Pickup', 'woocommerce' ), true );
```

Comment:
Would the ` ?? __( 'Local Pickup', 'woocommerce' )` be needed now we have a default? just in case the merchant save an empty string?